### DISCUSSION TITLE
NSFW

### DISCUSSION THREAD

**Discussion by trentschnee** [dailydotdev/daily#455] [Feature Requests]
Often, there are various blog article titles with profanity. As a web developer, I use daily.dev. However, when I want to share my screen, I have to make sure I'm using a different web browser that doesn't have dail.dev configured for my homepage.

**Comment by idoshamun** [comment]
Hi @trentschnee,

We're sorry you're experiencing this. Can you share some examples? I suspect it has to do with specific sources that we will have to deal with

**Reply by trentschnee** [reply]
Sure,  [here is an example](https://imgur.com/6s20Oeq). I mean it's a great article but sometimes my boss might be overlooking at I'd hate for him to tell me I can't use this as my homepage anymore because it contains profanity.

**Reply by idoshamun** [reply]
For now, the immediate action I can suggest is to block Hacker News. We look at what can be done in the future about explicit language titles.
